osu.Game.Tournament.Tests/Screens/TestSceneLadderEditorScreen.cs

```diff
@@ -1,22 +1,99 @@
 ﻿// Copyright (c) ppy Pty Ltd <contact@ppy.sh>. Licensed under the MIT Licence.
 // See the LICENCE file in the repository root for full licence text.
 
-using osu.Framework.Allocation;
+#nullable disable
```

Comment:
Please don't use `#nullable disable` in new code unless there's a very good reason to do so.